### PR TITLE
Merge changes from beta 19.8.0.1

### PR DESCRIPTION
**Pull request by mokagio**
This is a bit of an odd beta, because there are no new features or fixes in the diff.

You might recall that I had to followup the [code freeze PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/18494) with [a commit](https://github.com/wordpress-mobile/WordPress-iOS/pull/18494/commits/4a4126a1f1ad2df889a34539983327779909a5f2) to point WordPressKit to a new version but I wasn't able to ship a new beta for it because of a change on the GlotPress backend.

That change required [some work on the release toolkit](https://github.com/wordpress-mobile/release-toolkit/pull/361). With the change in (see the set and reset of the toolkit to its `trunk` branch in the commit history) we can now ship a new beta.

That's all to say: Other than for the localizations and `.xcconfig`, there's nothing else in this diff, _and that's expected._ I did not want to wait any longer to ship the beta.

- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [ ] ~~Diffs from the PRs that made it into this beta~~ – N.A.

### CODE REVIEW

File: WordPress/Resources/bg.lproj/Localizable.strings [hunk 1]
```diff
@@ -3212,6 +3215,12 @@
 /* Text for related post cell preview */
 "in \"Upgrade\"" = "в \"Обновяване\"";
 
+/* This text is used when the user is configuring the iOS widget to suggest them to select the site to configure the widget for */
+"ios-widget.gpCwrM" = "Select Site";
+
+/* This text is required by Apple to be part of the translations but is not shown to the users. You can use the English version verbatim without translation for this entry. */
+"ios-widget.tVvJ9c" = "Select Site Intent";
```
Comment (mokagio) [inline WordPress/Resources/bg.lproj/Localizable.strings:3222]: These should be in all the locales that didn't have them yet, and should stop ASC from sending us warning emails about missing translations for the Intent.

File: WordPress/WordPressIntents/bg.lproj/Sites.strings [hunk 2]
```diff
@@ -2,5 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <!--Warning: Auto-generated file, do not edit.-->
 <plist version="1.0">
-  <dict/>
+  <dict>
+    <key>gpCwrM</key>
+    <string>Select Site</string>
+    <key>tVvJ9c</key>
+    <string>Select Site Intent</string>
+  </dict>
```
Comment (AliSoftware) [inline WordPress/WordPressIntents/bg.lproj/Sites.strings:10]: 🎉 Looking forward to no more AppStore Connect warnings 😜

File: fastlane/jetpack_metadata/default/release_notes.txt [hunk 3]
```diff
@@ -1,11 +1,12 @@
-We made a small but mighty change to the block editor: the “Add Block” button is more visible when you first open the editor with no blocks selected. We also removed one of three error notifications when a media or audio block fails to upload—it felt like overkill.
+When you Follow Conversation in Reader post cards and post details, in-app notifications are now opt-out rather than opt-in.
 
-You’ll hear some accessibility tweaks in the VoiceOver experience when you’re rearranging menu items. Instructions and notices are clearer, and the menu hierarchy makes more sense out loud.
+We replaced the quick-start prompt when you log in as an existing user. Now you can tell us what you’d like to focus on and enable notifications for things like site traffic, new followers, and other milestones. Cue confetti.
```
Comment (AliSoftware) [inline fastlane/jetpack_metadata/default/release_notes.txt:3]: Cue 🎊 🎉

**Review by AliSoftware — APPROVED** [review]
- [ ] ~Changes from bugfix PRs which landed in the release branch~
   - None, but that's expected, as you mentioned in PR description
- [X] Version bump in `.xcconfig` files
- [X] Latest app and metadata translations pulled from Glotpress
   - No more error 429 🎉
   - I'm a bit surprised that none of the release notes have been translated in _any_ locale yet (all `release_notes.txt` files deleted but none replaced with a new copy, except `default/`), but ok 🤷‍♂️

**Comment by AliSoftware** [comment]
> I'm a bit surprised that none of the release notes have been translated in any locale yet

FWIW I checked in GlotPress and indeed none of the locale have a translation for `v19.8-whats-new`, so that matches what we see in the PR (I just thought that maybe by now at least one or two locale would have had a translation, hence my surprise; but apparently not just yet 🙃)

**Pull request merged**